inspector/loadavg.go

```diff
@@ -95,14 +103,41 @@ func (i *LoadAvg) Execute() {
 	}
 }
 
-//TODO: Windows Equivalents
-// of LoadAvg
+func (i *LoadAvgWin) Parse(output string) {
+	output = strings.ReplaceAll(output, "\r", "")
```

Comment:
Probably add some comments as to why we do the things we do. For context. Also you can paste the output format as a comment. I think we should do that for all of the Parse functions. For example under meminfo parse we could add a format 
```emTotal:        8148696 kB
MemFree:         6601804 kB
MemAvailable:    7366832 kB
Buffers:           91004 kB
Cached:           853700 kB
SwapCached:            0 kB
Active:           386584 kB
Inactive:        1010864 kB
Active(anon):       1016 kB
Inactive(anon):   441856 kB
Active(file):     385568 kB
Inactive(file):   569008 kB
Unevictable:           0 kB
Mlocked:               0 kB
SwapTotal:             0 kB
SwapFree:              0 kB
Dirty:               824 kB
Writeback:             0 kB
AnonPages:        452860 kB
Mapped:           349932 kB
Shmem:              1164 kB
KReclaimable:      63424 kB
Slab:              92756 kB
SReclaimable:      63424 kB
SUnreclaim:        29332 kB
KernelStack:        6480 kB
PageTables:         7032 kB
NFS_Unstable:          0 kB
Bounce:                0 kB
WritebackTmp:          0 kB
CommitLimit:     4074348 kB
Committed_AS:    4592968 kB
VmallocTotal:   34359738367 kB
VmallocUsed:       11104 kB
VmallocChunk:          0 kB
Percpu:             1320 kB
HardwareCorrupted:     0 kB
AnonHugePages:         0 kB
ShmemHugePages:        0 kB
ShmemPmdMapped:        0 kB
FileHugePages:         0 kB
FilePmdMapped:         0 kB
CmaTotal:              0 kB
CmaFree:               0 kB
HugePages_Total:       0
HugePages_Free:        0
HugePages_Rsvd:        0
HugePages_Surp:        0
Hugepagesize:       2048 kB
Hugetlb:               0 kB
DirectMap4k:       97088 kB
DirectMap2M:     4093952 kB
DirectMap1G:     6291456 kB
```